onionshare/web/web.py

```diff
@@ -60,10 +60,16 @@ class Web:
     REQUEST_OTHER = 13
     REQUEST_INVALID_PASSWORD = 14
 
-    def __init__(self, common, is_gui, mode="share"):
+    def __init__(self, common, is_gui, mode_settings, mode="share"):
+        """
+        tab_settings_get and tab_settings_set are getter and setter functions for tab settings
```

Comment:
I can't find these functions 'tab_settings_get' and 'tab_settings_set'?

Reply:
Good catch. I changed things to use `ModeSettings`, so I don't need that comment at all anymore